<launch>
  <node pkg="tf" type="static_transform_publisher" name="lidar_position" args="0. 0 0.45 0 0 0 base_link lidar_link 100">
  </node>
  <node name="hw_interface" pkg="ros_interface" type="hw_interface_node">
  </node>
  <node name="odom_node" pkg="ros_interface" type="odom_from_encoders_node">
  </node>
</launch>

